[Feat] Add info icon to components (#1144)
mckinsey/vizro: vizro-core/docs/pages/user-guides/container.md

        [![CollapsibleContainerGrid]][collapsiblecontainergrid]

## Add an info-icon

> The `description` argument enables you to add helpful context to your container by displaying an info icon next to its title. Hovering over the icon shows a tooltip with your provided text.

Reviewer: ```suggestion
The `description` argument enables you to add helpful context to your container by displaying an info icon next to its title. Hovering over the icon shows a tooltip with your chosen text.
```

> You can provide a string to use the default info icon, or pass a custom [`Tooltip`][vizro.models.Tooltip] model to define both a custom icon from the [Google Material Icons library](https://fonts.google.com/icons) and the description text.

Reviewer: ```suggestion
To use the default info icon, just provide a string of description text. To use a custom icon, pass a custom [`Tooltip`][vizro.models.Tooltip] model to use an icon from the [Google Material Icons library](https://fonts.google.com/icons) alongside your description text.
```

Reviewer 2: I'd probably add a code example here given that it's very short anyway? Something like below:

```To use the default info icon, just provide a string of description text. To use a custom icon, pass a custom [`Tooltip`][vizro.models.Tooltip] model to use an icon from the [Google Material Icons library](https://fonts.google.com/icons) alongside your description text e.g. description=Tooltip(XXXXX)```

!!! example "Container with info-icon"

    === "app.py"

>         ```{.python pycafe-link}

Reviewer 2: I think we may need to retake the screenshots to ensure they match the examples exactly. Since we haven’t included the Vizro logo in any of the examples so far, the screenshots should also not display the logo 😬
